pos_partner_deselection/doc/index.rst

```diff
@@ -1,25 +1,29 @@
-==================
- Deselect Partner
-==================
+===================
+Partner deselection
+===================
+
+Installation
+============
+
+* `Install <https://odoo-development.readthedocs.io/en/latest/odoo/usage/install-module.html>`__ this module in a usual way
 
 Configuration
 =============
 
 * Go to ``Point of Sale >> Configuration >> Point of Sale``
 
-  * Select a POS
-  * Click on ``[Edit]``
+  * Select a Point of Sale (POS)
+  * Click on ``[Edit]`` and then select  ``[Settings]``
```

Comment:
different flow either 13 or 16, but in the second option, anyway, there is no `Edit`

Reply:
fixed